File: DotNet-Solves/2176_A.py
for _ in range(int(input())):
    n = int(input())
    arr = list(map(int, input().split()))
    ans = 0
    seg_size = 0
    
    for i in range(1, n):
        ans += 0 if (arr[i] > max(arr[0: i])) else 1

    print(ans)

File: DotNet-Solves/2176_A.cs
using System;

var _ = int.TryParse(Console.ReadLine(), out int testCases);

foreach (var __ in Enumerable.Range(0, testCases))
{
    var n = int.Parse(Console.ReadLine()!);
    var arr = Console.ReadLine()!.Split().Select(int.Parse).ToArray();
    var ans = Enumerable.Range(1, n - 1).Sum(i => arr[i] < arr[0..i].Max() ? 1 : 0);
    Console.WriteLine(ans);   
}